JavaScript/Timer.js

```diff
@@ -0,0 +1,63 @@
+//this code is written to operationalize a timer with 3 main functions of Start, Stop and Reset.
```

Comment:
Use g.itignore to not include files such as .DS_Store (which is for your Mac), or `.idea` (which is for JetBrains products).

Reply:
made .gitignore and passed the .idea and .DS_Store in it.